book/tut2_spec_pd_gpd.ipynb

```diff
@@ -0,0 +1,1002 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "315a335b-1325-4319-aa06-cb1b239616b8",
+   "metadata": {},
+   "source": [
+    "# **Tutorial 2** - scientific Python ecosystem: `pandas` and `GeoPandas`\n",
+    "\n",
+    "In this tutorial we will learn using\n",
+    " 1. [**pandas**](https://pandas.pydata.org/docs/): **tabular data** stored in ``pandas.DataFrame``s\n",
+    " 2. [**geoandas**](https://geopandas.org/en/stable/docs.html): **aspatial data** stored in ``geopandas.GeoDataFrame``s\n",
```

Comment:
Is the final "s" a typo?

Reply:
No, I think, this is related to the Jupyter notebook itself.